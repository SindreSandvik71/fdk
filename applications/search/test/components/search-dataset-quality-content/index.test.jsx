import React from 'react';
import { shallow } from 'enzyme';
import DatasetQuality from '../../../src/components/search-dataset-quality-content';

let defaultProps;
let wrapper;

beforeEach(() => {
  defaultProps = {
    relevanceAnnotation: 'relevance',
    completenessAnnotation: 'comleteness',
    accuracyAnnotation: 'accuracy',
    availabilityAnnotations: 'availability'
  };
  wrapper = shallow(<DatasetQuality {...defaultProps} />);
});

test('should render DatasetQuality correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
