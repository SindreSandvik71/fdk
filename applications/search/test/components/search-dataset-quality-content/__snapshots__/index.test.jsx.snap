// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DatasetQuality correctly 1`] = `
<section>
  <div
    className="fdk-container-detail fdk-container-detail-header"
  >
    <i
      className="fa fa-star fdk-fa-left fdk-color-cta"
    />
    Kvalitet på innhold
  </div>
  <div
    className="fdk-container-detail"
  >
    <div>
      <h5>
        Relevans
      </h5>
      <p
        className="fdk-ingress fdk-margin-bottom-double"
      >
        relevance
      </p>
    </div>
    <div>
      <h5>
        Kompletthet
      </h5>
      <p
        className="fdk-ingress fdk-margin-bottom-double"
      >
        comleteness
      </p>
    </div>
    <div>
      <h5>
        Nøyaktighet
      </h5>
      <p
        className="fdk-ingress fdk-margin-bottom-double"
      >
        accuracy
      </p>
    </div>
    <div>
      <h5>
        Tilgjengelighet
      </h5>
      <p
        className="fdk-ingress fdk-margin-bottom-double"
      >
        availability
      </p>
    </div>
  </div>
</section>
`;
